_posts/2020-05-20-gsoc-students.md

```diff
@@ -0,0 +1,131 @@
+user/software signatures. The primary goal of [this
+project](https://summerofcode.withgoogle.com/projects/#5116604104310784)
+is for @hmacdope to refactor the existing TNG code into C++ to provide
+clarity and usability for GROMACS, other simulation packages and
+analysis tools. Thin FORTRAN and Python layers are also desirable to
+encourage widespread adoption and are a secondary goal of the
+project. An efficient and transferable implementation of the TNG
+format will represent a major step forward for the computational
+molecular sciences community, enabling easy storage and replication of
+simulations.
+
+This project is a collaboration with the [GROMACS][] developer team
+with @acmnpv from GROMACS serving as a co-mentor.
+
+Hugo MacDermott-Opeskin is a PhD student in computational chemistry at
+the Australian National University. His work focuses on studying
+membrane biophysics through molecular dynamics simulations coupled
+with enhanced sampling techniques.  Hugo can be found on github as
+[@hmacdope][hmacdope-github] and on twitter as
+[@hugomacdermott][hmacdope-twitter]. When not hard at work Hugo can
+be found running or mountain biking in the Canberra hills.
+
+Through GSoC Hugo aims to bring the TNG next generation trajectory
+format to the simulation community and he will document his experience at
+[his "Biophysics Bonanza" blog][hmacdope-blog].
+
+
+## Cédric Bouysset: From RDKit to the Universe and back
+
+<img
+src="https://cbouy.github.io/assets/img/photo-CV.jpeg"
+title="Cédric Bouysset" alt="Cédric Bouysset"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+The aim of the [RDKit interoperability
+project](https://summerofcode.withgoogle.com/projects/#6750913248624640)
+is to give MDAnalysis the ability to use [RDKit][]’s `Chem.Mol`
+structure as an input to an MDAnalysis `Universe`, but also to
+convert a `Universe` or `AtomGroup` to an RDKit `molecule`. RDKit is
+one of the most complete and one of the most commonly used
+chemoinformatics package, yet it lacks file readers for formats
+typically encountered in MD simulations. @cbouy will implement in
+MDAnalysis the ability to switch back and forth between a `Universe`
+and an RDKit `molecule` to perform typical chemoinformatics
+calculations and so add a lot of value to both packages.
+
+Cédric is a PhD student in molecular modelling at Université Côte
+D'Azur, France. His research aims to decipher the molecular basis
+of chemosensory perception (smell and taste) using computational
+tools. His day-to-day work includes; modelling bitter taste receptors,
+building machine-learning models to search for molecules with
+interesting olfactive or sapid properties, maintaining the website
+of the Global Consortium of Chemosensory Researchers, and a bit of
+teaching. In his free time he enjoys cooking and playing video games.
+Cédric can be found on github as [@cbouy][cbouy-github] and on twitter
```

Comment:
@IAlibay you don't need to explicitly link the GitHub – we have an extension that automatically expands the @-handle.